src/lib/transaction_logic/mina_transaction_logic.ml

```diff
@@ -789,7 +814,7 @@ module Make (L : Ledger_intf.S) :
       if should_update then L.apply_mask t.first_pass_ledger ~masked:ledger ;
       t
 
-    let second_pass_ledger { second_pass_ledger; _ } =
+    let _second_pass_ledger { second_pass_ledger; _ } =
```

Comment:
?

Reply:
Just finishing what you've started. You disabled taking first pass ledger, which broke the original txn logic. I've fixed it by initialising the local state with ledger directly, and disabled using second pass ledger.

Reply:
So now the original txn logic can be at least used for tests, in testing ledger.